src/reference/antora/modules/ROOT/pages/whats-new.adoc

```diff
@@ -44,3 +44,8 @@ See xref:redis.adoc#redis-lock-registry[Redis Lock Registry] for more informatio
 === Hazelcast Changes
 
 Previously deprecated classes in the `spring-integation-hazelcast` module, such as `LeaderInitiator`, `HazelcastMembershipListener`, `HazelcastLocalInstanceRegistrar` and `HazelcastLockRegistry`, are now removed due to not supported CP-subsystem in Hazelcast library for Open Source.
+
+[[x7.0-mqtt-changes]]
+=== MQTT Changes
+
+The `disconnectForcibly` method in `MqttPahoMessageDrivenChannelAdapter`, `Mqttv5PahoMessageDrivenChannelAdapter`, `Mqttv3ClientManager`, and `Mqttv5ClientManager` now supports `getQuiescentTimeout` in addition to the existing `disconnectCompletionTimeout`.
```

Comment:
That’s not quite true. That method had it supported for years in the target client. And in fact that is not a method of our classes. Please , try to be more specific. 
let’s say something like those classes now provide such an option which is propagated down to the respective client API.
Also pay attention to the typo of new property. It is really not `get…` copy/paste artifact I guess 😉
Also link to the target chapter , please.